### PR TITLE
ci(repo): migrate to github actions.

### PR DESCRIPTION
**Pull request by philippgualdi**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+          python-version: [3.6, 3.7, 3.8]
+          requirements-level: [min, pypi]
```
Comment (mb-wali) [inline .github/workflows/tests.yml:24]: Seeing the previous ```travis.yml``` file - this module needs some services - such as ES, DB in order to run the tests.
consider adding them in the github action yml file.

File: setup.py [hunk 2]
```diff
@@ -13,17 +13,17 @@
...
-    "invenio-i18n>=1.0.0",
+    "invenio-db>=1.0.6",
+    "invenio-i18n>=1.2.0",
     "invenio-indexer>=1.1.0",
-    "invenio-pidstore>=1.0.0",
-    "invenio-records>=1.0.0",
+    "invenio_search>=1.3.1",
     "pytest-invenio>=1.4.0",
+    "elasticsearch_dsl>=7.2.1",
+    "SQLAlchemy-Continuum>=1.3.11",
```
Comment (mb-wali) [inline setup.py:26]: what was the reason for changing these dependencies- cannot find anything in the description or in the commits.

**Review by mb-wali — CHANGES_REQUESTED** [review]
### Request changes
1. add ES, DB services to the GitHub action .yml file. See [reference](https://github.com/inveniosoftware/invenio-rdm-records/blob/95e8bbb3f5d3cec41e28e425622179c05573bd69/.github/workflows/tests.yml#L26)
2. update files ```run-tests.sh``` , ```pytest.ini```, ```CONTRIBUTING.rst``` and  ```README.rst``.  See [reference](https://github.com/tu-graz-library/invenio-theme-tugraz/pull/119)

File: .github/workflows/tests.yml (hunk 1 above)
Comment (philippgualdi) [reply .github/workflows/tests.yml:24]: On travis build we had the issue in the run-tests.sh https://github.com/tu-graz-library/invenio-marc21/blob/master/run-tests.sh#L10 docker-services-cli pulls already the elasticsearch container. 

Travis build log: https://travis-ci.com/github/tu-graz-library/invenio-marc21/jobs/449653590 on line 650

File: setup.py (hunk 2 above)
Comment (philippgualdi) [reply setup.py:26]: In the test/conftest.py we import InvenioIndexer which raises the  ModuleNotFoundError: No module named 'invenio_search' found, so I added explicit to install the packages for tests.

**Comment by philippgualdi** [comment]
I'll change the build script according to the  inveniosoftware  policy.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (mb-wali) [reply .github/workflows/tests.yml]: This was the issue from travis-ci - if you run again the job this will disappear.

**Review by mb-wali — APPROVED** [review]

**Pull request merged**